presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnectorFactory.java

```diff
@@ -60,4 +71,11 @@ public Connector create(String catalogName, Map<String, String> config, Connecto
             throw new RuntimeException(e);
         }
     }
+
+    public static class EmptyModule
```

Comment:
That's simply `binder -> {}`, inline

(or `com.google.inject.util.Modules.EMPTY_MODULE` as in the other place; i'd just use lambda)